### ISSUE TITLE
Question: RadialChart and classNames

### ISSUE THREAD

**Issue by andrewrocco** [uber/react-vis#444]
Is there a way currently to pass a custom `className` to an arc series item inside a `RadialChart`?

Perhaps something like:
```jsx
const radialData = [
  { angle: 1, label: 'Item 1', className: 'some-css-class' },
  { angle: 2, label: 'Item 2', className: 'another-css-class' },
];
```

Or is there already a system in place to accomplish the same thing another way? I am running these charts inside a CSS Modules setup and I need to be able to set very specific class names for theming. (CSS Modules currently work with the `RadialChart` component itself, just not anything inside it.)

Thanks!

**Comment by mcnuttandrew** [comment]
That functionality isn't currently built but it seems like it should be relatively easy to implement! I'm away from computer for a while, but would welcome a pr

**Comment by andrewrocco** [comment]
I can definitely take a stab at it. 💥

**Comment by mcnuttandrew** [comment]
Now that #445 is merged in, and v1.5.3 is cut, this issue should be resolved
